src/man/sssd.conf.5.xml

```diff
@@ -2468,8 +2468,9 @@ p11_uri = library-description=OpenSC%20smartcard%20framework;slot-id=2
                             name is requested.
                         </para>
                         <para>
-                            Default: FALSE (TRUE if default_domain_suffix is
-                            used)
+                            Default: FALSE (TRUE for trusted
+                            domain/sub-domains and if default_domain_suffix
```

Comment:
May be it should say `or` (not `and`)?

@sumit-bose, could you please take a look?

Reply:
@alexey-tikhonov yeah, it looks like 

> or

 will be better there. Let me make the changes.

Reply:
@alexey-tikhonov done.

Reply:
yes, `or` is better.